resources/js/components/pages/poll.vue

```diff
@@ -146,8 +162,11 @@ export default {
         const poll = response.data ? response.data.data : {};
 
         this.isAdmin = poll.is_admin;
-        // TODO: Make the wheel spin instant here
         this.isFinished = poll.chosen_item_id !== -1;
+        if (this.isFinished) {
+          this.isSpinning = true;
```

Comment:
isSpinning should be false if the poll was already finished. This if block can be deleted